<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseGeneModelColorWidget</class>
 <widget class="QWidget" name="ChooseGeneModelColorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="CDSLabel">
     <property name="text">
      <string>CDS color</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="CDSWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QWidget" name="CDSColorWidget" native="true"/>
      </item>
      <item>
       <widget class="QPushButton" name="CDSColorBtn">
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/icon/chooseColorIcon2.png</normalon>
         </iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="UTR5Label">
     <property name="text">
      <string>5'UTR color</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="UTR5Widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QWidget" name="UTR5ColorWidget" native="true"/>
      </item>
      <item>
       <widget class="QPushButton" name="UTR5ColorBtn">
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/icon/chooseColorIcon2.png</normalon>
         </iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="UTR3Label">
     <property name="text">
      <string>3'UTR color</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="UTR3Widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QWidget" name="UTR3ColorWidget" native="true"/>
      </item>
      <item>
       <widget class="QPushButton" name="UTR3ColorBtn">
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/icon/chooseColorIcon2.png</normalon>
         </iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
